app/views/buildings/new.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= provide(:title, "New building") %>
```

Comment:
How about `Add building` instead of `New`? I think this can be applied to all resource creation routes.